dm/pkg/terror/error_list.go

```diff
@@ -897,6 +899,8 @@ var (
 		"config '%s' regex pattern '%s' invalid, reason: %s", "Please check if params is correctly in the configuration file.")
 	ErrConfigOnlineDDLMistakeRegex = New(codeConfigOnlineDDLMistakeRegex, ClassConfig, ScopeInternal, LevelHigh,
 		"online ddl sql '%s' invalid, table %s fail to match '%s' online ddl regex", "Please update your `shadow-table-rules` or `trash-table-rules` in the configuration file.")
+	ErrOpenAPITaskConfigExists    = New(codeConfigOpenAPITaskConfigExists, ClassConfig, ScopeInternal, LevelLow, "the openapi task config  for '%s' already exists", "If you want to override it, pleaset use the overwite flag.")
+	ErrOpenAPITaskConfigNotExists = New(codeConfigOpenAPITaskConfigNotExists, ClassConfig, ScopeInternal, LevelLow, "the openapi task config  for '%s' not exists", "")
```

Comment:
```suggestion
	ErrOpenAPITaskConfigNotExists = New(codeConfigOpenAPITaskConfigNotExists, ClassConfig, ScopeInternal, LevelLow, "the openapi task config for '%s' does not exist", "")
```

Reply:
@Ehco1996 exists -> exist

Reply:
sorry  fixed in https://github.com/pingcap/ticdc/pull/3620/commits/0f81b94408124b2d3439141b28dd6befc289d340 and https://github.com/pingcap/ticdc/pull/3620/commits/385e8ad8448f5d06312a56a9a4cb72ed183a07f0